### PR TITLE
Added sample codes and updated README.md files for API:Rsd

### PR DESCRIPTION
**Pull request by tybanash**
Please review @brendajerop . Thank you.

### CODE REVIEW

**Comment by tybanash** [comment]
@brendajerop Please I am facing difficulties in imputing the response of API:Rsd in my sandbox in order to follow the format to be in JSON .  This is because the response i have is in xml. I also read from here https://en.wikipedia.org/wiki/Really_Simple_Discovery that RSD is an xml format. 
This is the link to my sandbox https://www.mediawiki.org/wiki/User:Awah_Nadege/Sandbox/API:Rsd.
In the sandbox the response overlaps out of its section. How can i fix this please?

File: javascript/rsd.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+//This file is autogenerated. See modules.json and autogenerator.py for details
```
Comment (brendajerop) [inline javascript/rsd.js]: @tybanash I see that this file is autogenerated. However, I cannot see where you have added your JSON schema in `modules.json` in this commit

**Comment by brendajerop** [comment]
> @brendajerop Please I am facing difficulties in imputing the response of API:Rsd in my sandbox in order to follow the format to be in JSON . This is because the response i have is in xml. I also read from here https://en.wikipedia.org/wiki/Really_Simple_Discovery that RSD is an xml format.
> This is the link to my sandbox https://www.mediawiki.org/wiki/User:Awah_Nadege/Sandbox/API:Rsd.
> In the sandbox the response overlaps out of its section. How can i fix this please?

@tybanash You're right that RSD is an xml format so there's no problem with your response :)

File: javascript/rsd.js (hunk 1 above)
Comment (tybanash) [reply javascript/rsd.js]: @brendajerop Oupps!!,  I will fix that right away, may be i deleted by error.

**Comment by tybanash** [comment]
I have sent another commit @brendajerop

**Comment by tybanash** [comment]
I have resolved the conflict as well.

**Comment by tybanash** [comment]
> > @brendajerop Please I am facing difficulties in imputing the response of API:Rsd in my sandbox in order to follow the format to be in JSON . This is because the response i have is in xml. I also read from here https://en.wikipedia.org/wiki/Really_Simple_Discovery that RSD is an xml format.
> > This is the link to my sandbox https://www.mediawiki.org/wiki/User:Awah_Nadege/Sandbox/API:Rsd.
> > In the sandbox the response overlaps out of its section. How can i fix this please?
> 
> @tybanash You're right that RSD is an xml format so there's no problem with your response :)

Okay thank you.

File: modules.json [hunk 2]
```diff
@@ -519,10 +519,10 @@
             "action": "query",
             "list": "blocks",
             "bklimit": "3",
-			"bkprop": "id|user|by|timestamp|expiry|reason|range|flags",
-			"format": "json"
+			      "bkprop": "id|user|by|timestamp|expiry|reason|range|flags",
```
Comment (brendajerop) [inline modules.json]: It seems you have made the indentation of the other schemas inconsistent?

File: modules.json [hunk 3]
```diff
@@ -582,16 +582,25 @@
...
+            "format": "json"
+        }
+    },
+    {
+        "filename": "rsd",
+        "docstring": "Demo of `Rsd` module: Get request to export an RSD schema.",
+        "endpoint": "https://en.wikipedia.org/w/api.php",
+        "params": {
+            "action": "rsd",
+			      "format": "json"
```
Comment (brendajerop) [inline modules.json]: Indentation. It's not usually visible at first glance, but check whether you are consistently using spaces or tabs in your schema. Mixing them is what results in the inconsistency.

**Comment by tybanash** [comment]
@brendajerop , I'm facing some issues. I ran git pull --rebase upstream master as you instructed and it gave me that
RSD ERROR 1
After i modified the indentation error on the modules.json file and did a git add modules.json . Whe i did git status, it showed :
nothing to commit, working tree clean
then i tried to do a git push origin branch and i gave me this error 
rsd error 2
I have tried to fix it with a git force push as i saw on one tutorial but it still didnt work. Please what do you suggest?

**Comment by brendajerop** [comment]
@tybanash Once you do a rebase, you'll have to force push. What error were you getting when you try to force push?

**Comment by tybanash** [comment]
I did the last commit "Fixed indentation" after working on Atom text editor and the same scenario occured. the indentation is perfect on my editor put when i push to github everything changes. 
Please see here 
Modules json

**Comment by brendajerop** [comment]
@tybanash You may merge the content in your sandbox with that of the original page :)

**Comment by brendajerop** [comment]
@tybanash A tip concerning the indentation. If you are using Sublime, always highlight your code to check whether you are mixing up tabs and spaces. Mixing them up is what brings about the inconsistency :)

**Comment by tybanash** [comment]
> @tybanash A tip concerning the indentation. If you are using Sublime, always highlight your code to check whether you are mixing up tabs and spaces. Mixing them up is what brings about the inconsistency :)

Well noted. Thanks alot @brendajerop .

**Comment by tybanash** [comment]
> @tybanash You may merge the content in your sandbox with that of the original page :)

Okay

**Comment by tybanash** [comment]
I have merged the changes @brendajerop .

**Pull request merged**